app/components/clinic/BgRangeSummary.js

```diff
@@ -17,23 +17,72 @@ import { space, shadows } from '../../themes/baseTheme';
 import { utils as vizUtils } from '@tidepool/viz';
 const { reshapeBgClassesToBgBounds, generateBgRangeLabels } = vizUtils.bg;
 
-export const BgRangeSummary = props => {
-  const { bgUnits, data, striped, targetRange, t, ...themeProps } = props;
-  const formattedBgUnits = bgUnits.replace(/l$/, 'L');
+const roundUp = (value, precision = 1) => {
+  let shift = 10 ** precision;
+  return Math.ceil(value * shift) / shift;
+};
 
-  const formatValue = (value) => {
-    let precision = 0;
-    const percentage = value * 100;
+const roundDown = (value, precision = 1) => {
+  let shift = 10 ** precision;
+  return Math.floor(value * shift) / shift;
+};
+
+export const formatValue = (inputValue, key) => {
+  let precision = 0;
+  let percentage = inputValue * 100;
 
-    // We want to show extra precision on very small percentages so that we avoid showing 0%
-    // when there is some data there.
-    if (percentage > 0 && percentage < 0.5) {
-      precision = percentage < 0.05 ? 2 : 1;
-    }
+  // We want to show extra precision on very small percentages so that we avoid showing 0%
+  // when there is some data there.
+  if (percentage > 0 && percentage < 0.5) {
+    precision = percentage < 0.05 ? 2 : 1;
+  }
 
-    return format(`.${precision}f`)(percentage);
+  switch (key) {
+    case 'veryLow':
+      if (percentage > 0 && percentage < 0.5) {
+        precision = 2;
+        percentage = roundUp(percentage, precision);
+      }
+      if (percentage > 0.5 && percentage < 1) {
```

Comment:
I realize what you have matches the done criteria of the ticket, but, would this be better as:
```suggestion
      if (percentage >= 0.5 && percentage < 1) {
```
If I'm tracking this correctly, a `percentage` value of _exactly_ `0.5` would end up with a precision of `0`, and would be formatted as `0` (I don't think `d3-format` rounds for the`f` notation)

If this is correct, we should also update the done criteria of the ticket to suit.

Reply:
Ah, yeah - good catch. I'll make the adjustment and add a unit test for that case.

Reply:
@clintonium-119 added in the unit test, adjusted the boundary condition and updated the done criteria 👍🏼